save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestExecutionService.kt

```diff
@@ -82,6 +87,36 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
     internal fun getTestExecutionsCount(executionId: Long) = testExecutionRepository
         .countByExecutionId(executionId)
 
+    /**
+     * @param projectId
+     */
+    internal fun deleteTestExecutionWithProjectId(projectId: Long?) {
+        val testExecutions = testExecutionRepository.findAll { root, cq, cb ->
```

Comment:
Same, may be easier and more readable with spring-data method naming and runtime proxies

Reply:
Yes. Fixed